.github/workflows/create_release.yml

```diff
@@ -0,0 +1,58 @@
+name: "🚀 Release on merge"
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  create_release:
+    if: github.event.pull_request.merged == true
+    name: "Create Release"
+    runs-on: ubuntu-latest
+    
+    steps:
+    # Checking out the current branch
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    # Getting the description for the pull request that was merged
+    - name: Fetch Pull Request Details
+      run: |
+        SOURCE_BRANCH=$(jq --raw-output .pull_request.head.ref "$GITHUB_EVENT_PATH")
+        if [[ "$SOURCE_BRANCH" != release/* ]]; then
+          echo "Source branch does not match pattern. Exiting."
+          exit 1
+        fi
+        
+        VERSION_NUMBER=${SOURCE_BRANCH#release/}
+        if [[ ! "$VERSION_NUMBER" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+          echo "SUFFIX does not match semantic versioning format. Exiting."
```

Comment:
This regex doesn't match 5.14, not a big deal but still looks like a valid version to me

Reply:
the PR creation script always has the full semantic version, so should be fine I guess?